components/molecules/m-update-password.vue

```diff
@@ -2,7 +2,9 @@
   <div class="m-update-password">
     <p class="message">
       {{ $t('If you want to change the password to access your account, enter the following information:') }}<br>
-      {{ $t('Your current email address is') }} <span class="message__label">{{ currentUser.email }}</span>
+      <template v-if="currentUser">
+        {{ $t('Your current email address is') }} <span class="message__label">{{ currentUser ? currentUser.email : '' }}</span>
```

Comment:
if you have `v-if="currentUser"` above then it's not necessary to do `currentUser ? currentUser.email : ''` It will always has `currentUser.email`

Reply:
Right